caffe2/python/optimizer.py

```diff
@@ -140,7 +139,7 @@ def _run(self, net, param_init_net, param_info):
 
         if self.momentum > 0:
             momentum_data = param_init_net.ConstantFill(
-                param, str(param) + "_momentum", value=0.)
+                [param], param + '_momentum', value=0.)
```

Comment:
so do you need this change in your diff ? isn't it equivalent to what it was before?